advocate_europe/assets/scss/core/_bootstrap-variables.scss

```diff
@@ -7,16 +7,18 @@ $bootstrap-sass-asset-helper: false !default;
 $footer-height: 120px;
 $footer-screen-sm-hight: 185px;
 $ae-light-gray: #f3f3f3;
+$ae-lighter-gray: #fafafa;
 $ae-light-gray-shadow: #ccc;
 $ae-light-gray-darker: #e6e6e6;
 $ae-middle-gray: #999;
 $ae-dark-gray: #d8d8d8;
 $ae-font-size-extra-large: 48px;
 $ae-brand-orange: #fcb813;
+$ae-brand-green: #73be46;
+$ae-brand-red: #db3918;
```

Comment:
We already have $brand-danger and $brand-success, which usally maps to red and green. We want probably use them. If @damusp wants to we should change them, since they also affect a lot off different places.